docs/guides/dapis/subscribing-managed-dapis/index.md

```diff
@@ -0,0 +1,121 @@
+using [Airnode's](/explore/airnode/what-is-airnode.html) signed data. It
+utilises first-party oracles, enabling data to be served directly from reputable
+API providers via the Airnode. Unlike self-funded dAPIs, API3 manages the gas
+costs and the availability of the Managed dAPIs. This guide will walk you
+through the process of subscribing to a Managed dAPI.
+
+::: info Managed dAPIs
+
+If you are planning to use your dAPI for a long period of time or if you are
+planning to use your dAPI in a production environment, we recommend that you use
+Managed dAPIs provided by the API3 DAO. This way, you can be sure that your dAPI
+will be available to your users at all times.
+
+:::
+
+## API3 Market
+
+The [API3 Market<ExternalLinkImage/>](https://market.api3.org) provides all the
+tooling to seamlessly access dAPI services. For self-funded data feeds this
+end-to-end process consists of:
+
+- [Exploring, selecting and configuring your dAPI](/guides/dapis/subscribing-managed-dapis/#exploring-selecting-and-configuring-your-dapi)
+  <br/>
+- [Placing an order for a Managed dAPI Service](/guides/dapis/subscribing-managed-dapis/#placing-an-order-for-a-managed-dapi-service)
+  <br/>
+- [Waiting for the dAPI to get updated](/guides/dapis/subscribing-managed-dapis/#waiting-for-the-dapi-to-get-updated)
+  <br/>
+
+## Exploring, selecting and configuring your dAPI
+
+The [API3 Market<ExternalLinkImage/>](market.api3.org/) provides a list of all
+the dAPIs available across multiple chains including testnets. You can filter
+the list by chains and data providers. You can also search for a specific dAPI
+by name. Once selected you will land on the details page where you can find more
+information about the dAPI.
+
+Once you have selected your dAPI, you will then be presented with an option to
+choose from either **Managed** or **Self-funded**. Select Managed dAPIs.
+
+<img src="../../assets/images/managed_1.png" style="width:60%"/>
+
+Managed dAPIs gives you an option to configure the dAPI's
+[devation threshold](/reference/dapis/understand/deviations) and
+[heartbeat](/reference/dapis/understand/deviations.html#heartbeat). For each
+Managed feed, you will have the following options to choose from:
+
+| Deviation | Heartbeat |
+| --------- | --------- |
+| 0.25%     | 2 minutes |
+| 0.25%     | 24 hours  |
+| 0.5%      | 24 hours  |
+| 1%        | 24 hours  |
+
+_[Click here](/reference/dapis/understand/deviations.html#update-interval) to
+read more if you are unsure what suits your dApp_
+
+Check the final price and time frame for the selected configuration. If you are
+happy with the it, select **Add to Cart**. You can also add other dAPIs on the
+same nework to your cart.
+
+## Placing an order for a Managed dAPI Service
+
+Once you have added all the dAPIs, you can proceed to checkout. You will be
+presented with a summary of your order. If you have multiple dAPIs in your cart,
+you'll have to pay for them individually. If you are happy with the order,
```

Comment:
> Once you have added all the dAPIs, you can proceed to checkout. You will be
presented with a summary of your order. If you have multiple dAPIs in your cart,
you'll have to pay for them individually.

do you have to pay for them individually? I'm not sure myself

> If you are happy with the order,

already used 'if you are happy', instead say something like `to proceed with the order`